include/dbus_utility.hpp

```diff
@@ -83,6 +83,9 @@ using DBusInterfacesMap =
 using ManagedObjectType =
     std::vector<std::pair<sdbusplus::message::object_path, DBusInterfacesMap>>;
 
+// List of interfaces
+using InterfaceList = std::vector<std::string>;
+
```

Comment:
Since the second of `MapperServiceMap` is a interfaceList, we can also update `MapperServiceMap`.

```
// List of interfaces
using InterfaceList = std::vector<std::string>;

// Map of service name to list of interfaces
using MapperServiceMap = std::vector<std::pair<std::string, InterfaceList>;
```